src/renderer/features/governance/utils/getLocksForAccount.ts

```diff
@@ -0,0 +1,14 @@
+import { type BN, BN_ZERO } from '@polkadot/util';
+
+import { toAddress } from '@/shared/lib/utils';
+import { type Address, type TrackId } from '@shared/core';
+
+export const getLocksForAccount = (
```

Comment:
I think it's better to calculate address outside and rename method `getLocksForAddress`